src/util.js

```diff
@@ -41,6 +41,21 @@ Util.isIFrame = function() {
   }
 };
 
+// Get a unique, incrementing Id value for any object on the page.
+Util.getUniqueId = (function(prefix) {
+  var i = Math.floor(Math.random() * 999) + 100;
+  var pfx = prefix || '';
+  function inc(pfx) {
+    if (!pfx) {
+      pfx = '';
+    } else {
+      pfx += '-';
+    }
+    return pfx + i++;
+  }
+  return inc;
+})();
```

Comment:
Another objection is to possible collisions. Could you switch to using UUIDs? 

http://stackoverflow.com/questions/105034/create-guid-uuid-in-javascript